pxr/usdImaging/usdviewq/appController.py

```diff
@@ -2804,9 +2826,13 @@ def _getSaveFileName(self, caption, recommendedFilename):
         
         return saveName
 
+    def _getRecommendedFilenamePrefix(self):
+        return (self._parserData.usdFile.rsplit('.', 1)[0]
+                if self._parserData.usdFile
+                else 'new_file')
+
     def _saveOverridesAs(self):
-        recommendedFilename = self._parserData.usdFile.rsplit('.', 1)[0]
-        recommendedFilename += '_overrides.usd'
+        recommendedFilename = self._getRecommendedFilenamePrefix() + '_overrides.usd'
```

Comment:
Below at `os.path.samefile(saveName, self._parserData.usdFile)`: 

With an empty stage, if we want to save overrides to an existing file, `self._parserData.usdFile` will be None and an error occurs.